Makefile

```diff
@@ -90,5 +90,12 @@ check-inside-openshift: service worker test_image
 check-inside-openshift-zuul: test_image
 	ANSIBLE_STDOUT_CALLBACK=debug $(AP) files/check-inside-openshift.yaml
 
+setup-toolbox:
+	dnf install -y ansible
```

Comment:
I am not sure if it is necessary. We don't do that elsewhere.

Reply:
fedora-toolbox image doesn't have ansible preinstalled

Reply:
Aha, this is not run on the host? Then I would suggest some better name.

Reply:
what name do you suggest? :)

I could also move ansible installation into the playbook and add check for `/run/.envrc` to make sure we're running in container / toolbox

Reply:
`setup-inside-toolbox`/`install-dependecies-into-toolbox`/`dependencies-inside-toolbox`/..

I now understand how your name makes sense but was confused originally if we are setting-up the toolbox or making setup inside toolbox.